datasets/pragmeval/pragmeval.py

```diff
@@ -0,0 +1,430 @@
+        url = "https://www.aclweb.org/anthology/E17-2092",
+        pages = "578--585",
+        abstract = "We describe EmoBank, a corpus of 10k English sentences balancing multiple genres, which we annotated with dimensional emotion metadata in the Valence-Arousal-Dominance (VAD) representation format. EmoBank excels with a bi-perspectival and bi-representational design. On the one hand, we distinguish between writer{'}s and reader{'}s emotions, on the other hand, a subset of the corpus complements dimensional VAD annotations with categorical ones based on Basic Emotions. We find evidence for the supremacy of the reader{'}s perspective in terms of IAA and rating intensity, and achieve close-to-human performance when mapping between dimensional and categorical formats.",
+    }
+    """,
+}
+
+
+def get_labels(task):
+    folder_name = TASK_TO_FOLDER[task]
+    label_classes = requests.get(
+        f"https://raw.githubusercontent.com/disceval/DiscEval/master/disceval/{folder_name}/labels"
+    ).text.split("\n")
+    label_classes = [x.strip() for x in label_classes if x.strip()]
+    return label_classes
+
+
+class PragmEvalConfig(datasets.BuilderConfig):
+    """BuilderConfig for PragmEval."""
+
+    def __init__(
+        self,
+        text_features,
+        label_classes=None,
+        process_label=lambda x: x,
+        **kwargs,
+    ):
+        """BuilderConfig for PragmEval.
+        Args:
+          text_features: `dict[string, string]`, map from the name of the feature
+            dict for each text field to the name of the column in the tsv file
+          label_column: `string`, name of the column in the tsv file corresponding
+            to the label
+          data_url: `string`, url to download the zip file from
+          data_dir: `string`, the path to the folder containing the tsv files in the
+            downloaded zip
+          citation: `string`, citation for the data set
+          url: `string`, url for information about the data set
+          label_classes: `list[string]`, the list of classes if the label is
+            categorical. If not provided, then the label will be of type
+            `datasets.Value('float32')`.
+          process_label: `Function[string, any]`, function  taking in the raw value
+            of the label and processing it to the form required by the label feature
+          **kwargs: keyword arguments forwarded to super.
+        """
+        super(PragmEvalConfig, self).__init__(version=datasets.Version("1.0.0", ""), **kwargs)
+        self.text_features = text_features
+        self.label_column = "label"
+        self.label_classes = get_labels(self.name)
+        self.data_url = DATA_URL
+        self.data_dir = os.path.join("pragmeval", self.name)
+        self.citation = textwrap.dedent(CITATION_DICT[self.name.split("-")[0]])
+        self.process_label = process_label
+        self.description = ""
+        self.url = ""
+
+
+class PragmEval(datasets.GeneratorBasedBuilder):
```

Comment:
```suggestion
class Pragmeval(datasets.GeneratorBasedBuilder):
```

The class name needs to be the camel case version of the script name `pragmeval`